src/main/java/io/lenses/connect/smt/header/TimestampConverter.java

```diff
@@ -442,14 +455,21 @@ public void configure(Map<String, ?> configs) {
 
     final String unixPrecision = simpleConfig.getString(UNIX_PRECISION_CONFIG);
 
+    final String targetTimeZone = simpleConfig.getString(TARGET_TIMEZONE_CONFIG);
+    TimeZone timeZone = TimeZone.getTimeZone(targetTimeZone);
+    if (timeZone == null) {
```

Comment:
Please double-check the docs but it will never be null: I believe it returns GMT if passed gibberish and NPE if passed null (which I believe is not the case as you have default value for config)

Reply:
Thank you. What a consistent library behaviour. It does make sense to return GTM when the value is nonsensical! [insert sarcasm]

Reply:
You haven't seen date and time management in Java 6, that was another Vietnam 😂